public/content/de/welcome/welcome-to-d.md

```diff
@@ -1,12 +1,60 @@
 # Willkommen zu D
 
+Glückwunsch! Du hast es zu der interaktiven Tour der Programmiersprache D geschafft.
+
+{{#dmanmobile}}
+
+Diese Tour wird dir einen Überblick über diese __mächtige__ und __expressive__
+Sprache, welche direkt zu __effizientem__, __nativem__ Maschinencode kompiliert.
+
+{{/dmanmobile}}
+
+### Was ist D?
+
+D ist die Akkumulation von Jahrzehnten an Erfahrung im Bau von Kompilierer
```

Comment:
funny for the noun the english form is more popular

http://www.duden.de/rechtschreibung/Compiler

whereas the verb uses the german form

http://www.duden.de/rechtschreibung/kompilieren